packages/turf-flatten/index.d.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference types="geojson" />
+
+type Feature = GeoJSON.Feature<any>;
+type Features = GeoJSON.FeatureCollection<any>;
+
+/**
+ * http://turfjs.org/docs.html#flatten
+ */
+declare function flatten(features: Feature | Features);
```

Comment:
The function has to return an output (that's why the Travis CI build is failing).

Also, typically you try to use the same `@param` names, (features => input)

```javascript
declare function flatten(input: Feature | Features): Features;
```

This function can be a bit more complex others since the outputs change based on their inputs, you can look at another Turf module that has a complex `index.d.ts`. (https://github.com/Turfjs/turf/blob/master/packages/turf-random/index.d.ts)

**FYI: I can take care of adding that after we merge.